editoast/src/views/infra/mod.rs

```diff
@@ -374,15 +375,23 @@ async fn rename(
 }
 
 /// Return the railjson list of switch types
+#[utoipa::path(
+    tag = "infra",
+    params(InfraIdParam),
+    responses(
+        (status = 200, description = "A list of switch types", body = Vec<SwitchType>,  example = json!({"id": "Point", "ports": ["LEFT", "RIGHT", "BASE"], "groups": { "LEFT": { "src": "BASE", "dst": "LEFT" }, "RIGHT": { "src": "BASE", "dst": "RIGHT" }}})),
```

Comment:
Better move the example to the `SwitchType` schema directly.

Reply:
Done.